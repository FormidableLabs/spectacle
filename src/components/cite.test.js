import React from 'react';
import { mount } from 'enzyme';
import { mountToJson } from 'enzyme-to-json';
import Cite from './cite';

describe('<Cite />', () => {
  test('should render correctly.', () => {
    const context = { styles: { components: { cite: { color: '#ee0' } } } };
    const wrapper = mount(<Cite>Someone</Cite>, { context });
    expect(mountToJson(wrapper)).toMatchSnapshot();
  });
});
