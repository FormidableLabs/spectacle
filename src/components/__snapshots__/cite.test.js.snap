// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Cite /> should render correctly. 1`] = `
<Cite>
  <Styled(cite)
    className={undefined}
    styles={
      Array [
        Object {
          "color": "#ee0",
        },
        Object {},
        Object {},
        undefined,
      ]
    }
  >
    <cite
      className="css-1j6fsf6"
    >
      - 
      Someone
    </cite>
  </Styled(cite)>
</Cite>
`;
